code/modules/uplink/uplink_items/suits.dm

```diff
@@ -64,6 +64,12 @@
 	item = /obj/item/mod/module/noslip
 	cost = 2
 
+/datum/uplink_item/suits/shock_absorber
+	name = "MODsuit Shock-Absorber Module"
+	desc = "A MODsuit module preventing the user from getting knocked down by batons"
+	item = /obj/item/mod/module/shock_absorber
+	cost = 4
```

Comment:
maybe drop this to 2-3. you already have to spend tc on the suit itself and noslips are 2